parser-utils/src/query.rs

```diff
@@ -11,7 +11,6 @@
 extern crate combine;
```

Comment:
Moved to lib.rs, my logic being that `matches_plain_symbol` uses `satisfy_unwrap` so it's good to have it close by.